src/bosh-director/lib/bosh/director/api/controllers/disks_controller.rb

```diff
@@ -20,7 +20,12 @@ class DisksController < BaseController
 
       delete '/:orphan_disk_cid' do
         job_queue = JobQueue.new
-        task = Bosh::Director::Jobs::DeleteOrphanDisks.enqueue(current_user, [params[:orphan_disk_cid]], job_queue)
+        orphan_param = params['orphan']
+        if orphan_param.nil? || orphan_param == 'false'
+          task = Bosh::Director::Jobs::DeleteOrphanDisks.enqueue(current_user, [params[:orphan_disk_cid]], job_queue)
+        else
+          task = Bosh::Director::Jobs::OrphanDiskJob.enqueue(current_user, params[:orphan_disk_cid], job_queue)
+        end
```

Comment:
I disagree with using `DELETE` for both deletion, but also non-deletion behavior. Technically I think this endpoint should delete the CID, regardless of orphaning status (even if we error for active disks).

Can we consider a `post /:orphan_disk_cid/orphan` endpoint instead?